### PR TITLE
Feat/capi template update

### PR DESCRIPTION
**Pull request by garloff**
Update external cluster-template to use alpha5 APIversions and adjust naming of resources to upstream.
However, keep PREFIX consistently in front of OpenStack resources where we can.
Network, Subnets and Loadbalancers created from CAPO unfortunately don't comply.
Probably, we'll need to reflect this in the cleanup script.

### CODE REVIEW

**Comment by garloff** [comment]
Note to reviewer:
I applied the patches to the branch feat/capi-template-update which I accidentially branched off feat/sign-certs.
So please review the diffs against between those two branches.
Merge strategy is: Land feat/sign-certs and then land feat/capi-template-update in main.
If feat/sign-certs is rejected, we can of course still cherry-pick the capi-template-update patches into a separate branch and merge into main.

**Review by matfechner — COMMENTED** [review]
look to my comment, the other things looks good to me

File: terraform/Makefile [hunk 1]
```diff
@@ -59,7 +60,7 @@ mycloud: environments/environment-$(ENVIRONMENT).tfvars
 
 gitchk:
 	@git diff -r origin/$(GITBRANCH) > git.diff
-	@if test -s git.diff; then echo "WARN: Local changes won't be used on mgmtcluster. Commit and push them"; cat git.diff; fi
+	@if test -s git.diff; then echo "WARN: Local changes won't be u$(SED) on mgmtcluster. Commit and push them"; cat git.diff; fi
```
[the remark this answers is not in the document]
Comment (garloff) [reply terraform/Makefile]: Good spotting, thanks, will fix.

**Comment by garloff** [comment]
I guess, I should do a rebase (and push --force?) to make this reviewable?

**Comment by garloff** [comment]
Superceded by PR #262.

**Pull request closed without merging**